libyang/context.py

```diff
@@ -133,12 +133,11 @@ def get_module_data(
             Tuple of format string and YANG (sub)module schema
         """
         if self._module_data_clb is None:
-            return "", None
-        fmt_str, module_data = self._module_data_clb(
-            mod_name, mod_rev, submod_name, submod_rev
-        )
-        if module_data is None:
-            return fmt_str, None
+            return None
```

Comment:
It made sense to return "", None, do you really want to change it?

Reply:
According to `get_module_data() -> Optional[Tuple[str, str]]:`  it shall return either None or Tuple of strings. So it was a invalid return value.